cypher/cypher-docs/src/test/scala/org/neo4j/cypher/docgen/SchemaIndexTest.scala

```diff
@@ -39,8 +39,8 @@ class SchemaIndexTest extends DocumentingTestBase with QueryStatisticsTestSuppor
   )
 
   override val properties = Map(
-    "andres" -> Map("firstname" -> "Andres", "surname" -> "Taylor"),
-    "mark" -> Map("firstname" -> "Mark", "surname" -> "Needham")
+    "andres" -> Map("firstname" -> "Andres", "surname" -> "Taylor", "age" -> 40, "country" -> "Sweden"),
```

Comment:
I have a spatial docs PR in development which also adds to this, with a normal range query. I used 'highScore' instead of 'age' because I felt like I should not expose personal information like age about real people like Andres and Mark. I then used this for a min<n.prop<max type triple query, and then showed the same this for spatial to demonstrate a bounding-box query.

We should merge our two efforts here.

Reply:
I think age and co are good properties to use from a pedagogical point of view. However, we should definitely get better graph examples! This is something I've raised with the Docs Team - it needs to be done across the board.

Reply:
How do we deal with this case then? Invent new anonymous names? Get permission from Mark and Andres to publish their ages?